samples/nrf9160/multicell_location/prj.conf

```diff
@@ -56,7 +57,7 @@ CONFIG_DK_LIBRARY=y
 # Heap and stacks
 CONFIG_MAIN_STACK_SIZE=4096
 # The AT parser, link controller, and nRF Cloud (if enabled) libraries use the heap
-CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_HEAP_MEM_POOL_SIZE=8192
```

Comment:
We discussed about removing `CONFIG_MULTICELL_LOCATION_MAX_NEIGHBORS` with @trantanen. It seems that it's actually only used with HERE, there's no such limitation with nRF Cloud. With HERE it is used to calculate the size of the buffers reserved.

LTE Link Control has option `CONFIG_LTE_NEIGHBOR_CELLS_MAX`, which configures the maximum number of neighbor cell measurements returned. Maybe we could use that in here_integration.c, because the measurements are coming from LTE LC. Another option would be to always allocate big enough buffers for 17 neighbor cells, because that's the maximum the modem can return.

Reply:
i think that would be good for instructional purposes.

Reply:
Done.